docs/source/models/intervention_models/neonatal/antibiotics_intervention.rst

```diff
@@ -0,0 +1,131 @@
+
+We will then use the conditional probabilities for simulants with and without access to determine which simulants die from sepsis.
+
+While we are searching the literature, we will use a placeholder relative risk of 0.50 (95% CI 0.25-0.75) of sepsis mortality for neonates with access to antibiotics.
+
+
+Calibration Strategy
+--------------------
+
+The following decision tree shows all of the paths from delivery facility choice to antibiotics availability.  Distinct paths in the tree correspond to disjoint events, which we can sum over to find the population probability of sepsis mortality.  The goal here is to find internally consistent probabilities of sepsis mortality for the subpopulations with and without access to antibiotics, so that the baseline scenario can track who has access to antibiotics and still match the baseline sepsis mortality rate.
+
+.. graphviz::
+
+    digraph antibiotics {
+        rankdir = LR;
+        facility [label="Facility type"]
+        home [label="p_sepsis_without_antibiotics"]
+        BEmONC [label="antibiotics"]
+        CEmONC [label="antibiotics"]
+        BEmONC_wo [label="p_sepsis_without_antibiotics"] 
+        BEmONC_w [label="p_sepsis_with_antibiotics"]
+        CEmONC_wo [label="p_sepsis_without_antibiotics"] 
+        CEmONC_w [label="p_sepsis_with_antibiotics"]
+
+        facility -> home  [label = "home birth"]
+        facility -> BEmONC  [label = "BEmONC"]
+        facility -> CEmONC  [label = "CEmONC"]
+
+        BEmONC -> BEmONC_w  [label = "available"]
+        BEmONC -> BEmONC_wo  [label = "unavailable"]
+
+        CEmONC -> CEmONC_w  [label = "available"]
+        CEmONC -> CEmONC_wo  [label = "unavailable"]
+    }
+
+.. math::
+    \begin{align*}
+        p(\text{sepsis}) 
+        &= \sum_{\text{paths without antibiotics}} p(\text{path})\cdot p(\text{sepsis_wo})\\
+        &+ \sum_{\text{paths with antibiotics}} p(\text{path})\cdot p(\text{sepsis_w})\\[.1in]
+        p(\text{sepsis_w}) &= \text{RR}_\text{antibiotics} \cdot p(\text{sepsis_wo})
+    \end{align*}
+
+where :math:`p(\text{sepsis})` is the probability of dying from sepsis in the general population, and :math:`p(\text{sepsis_w})` and :math:`p(\text{sepsis_wo})` are the probability of dying from sepsis in setting with and without access to antibiotics.  For each path through the decision tree, :math:`p(\text{path})` is the probability of that path; for example the path that includes the edges labeled BEmONC and unavailable occurs with probability that the birth is in a BEmONC facility times the probability that the facility has antibiotics available.
+
+When we fill in the location-specific values for delivery facility rates, antibiotics coverage, relative risk of mortality with antibiotics access, and mortality probability (which is also age-specific), this becomes a system of two linear equations have two unknown (p_RDS_w and p_RDS_wo), which we can solve analytically.
```

Comment:
```suggestion
When we fill in the location-specific values for delivery facility rates, antibiotics coverage, relative risk of mortality with antibiotics access, and mortality probability (which is also age-specific), this becomes a system of two linear equations have two unknown (p_sepsis_w and p_sepsis_wo), which we can solve analytically.
```

Reply:
As with CPAP, I think we need to rewrite this to describe how to calculate a PAF and apply the RR to the CSMR.